satpy/readers/viirs_active_fires.py

```diff
@@ -0,0 +1,68 @@
+from satpy.readers.netcdf_utils import NetCDF4FileHandler
+from satpy.readers.file_handlers import BaseFileHandler
+import os
+import dask.dataframe as dd
+import xarray as xr
+
+
+class VIIRSActiveFiresFileHandler(NetCDF4FileHandler):
+
+    def get_dataset(self, dsid, dsinfo):
```

Comment:
Docstrings for the class, this method, and maybe the properties would be nice. The `get_dataset` should include google style `Args:` documentation.